netlify.toml

```diff
@@ -1,2 +1,2 @@
 [build]
-  ignore = "exit 1"
```

Comment:
This looks like it may have reverted the commit on main? 🤔 I see the commit that introduced this in your branch which seems off.